README.md

```diff
@@ -33,7 +33,7 @@ This module provides Stackdriver Trace support for Node.js applications. [Stackd
 
 3. Include and start the library *as the very first action in your application*:
 
-        require('@google/cloud-trace').start();
+        var agent = require('@google/cloud-trace')().startAgent();
```

Comment:
Just a reminder that the `start` to `startAgent` rename will need to be undone in a follow-on.